client/hack/README.md

```diff
@@ -0,0 +1,81 @@
+# Scripts User Guide
+
+This README documents:
+* What update-crd.sh and update-generated-code.sh do
+* When and how to use them
+
+## update-generated-code.sh
+
+This is the script to update clientset/informers/listers and API deepcopy code using [code-generator](https://github.com/kubernetes/code-generator).
```

Comment:
Should include these pre-requisites before running update-generated-code.sh:

1.  Ensure external-snapshotter repo is at ~/go/src/github.com/kubernetes-csi/external-snapshotter
2. git clone https://github.com/kubernetes/code-generator.git under ~/go/src/k8s.io/code-generator/ and git checkout v0.19.0-rc.2"
3. GOPATH=~/go
4. Ensure this exists ${GOPATH}/src/k8s.io/code-generator/generate-groups.sh